mlir-tensorrt/test/Dialect/TensorRTRuntime/invalid.mlir

```diff
@@ -31,3 +31,30 @@ func.func @enqueue_host_tensor_el_type(%arg0: !trtrt.context, %arg1: !cuda.strea
     : (tensor<1xf32>, tensor<1xi32>) -> (tensor<1xf32>)
   return %0 : tensor<1xf32>
 }
+
+// -----
+
+// CHECK-LABEL: @enqueue_alloc_no_results
+func.func @enqueue_alloc_no_results(%ctx: !trtrt.context, %stream: !cuda.stream, %arg0: tensor<1x3x256x256xf32>) {
+  // expected-error @+1 {{cannot name an operation with no results}}
```

Comment:
This isn't testing your error message in the `trtrt.enqueue_alloc` verifier.  Just remove the `%result = ` to test your code. this message is saying that there's a `%result` with no type attached, it's from the MLIR core parser.

Reply:
Thanks for pointing this out. Your suggestion worked and now I can test the actual error from the check I added.